doc/requirements.txt

```diff
@@ -18,5 +18,6 @@ sphinxcontrib-bibtex==0.4.2
 tensorflow==2.5.1
 tensornetwork==0.3
 toml
-torch==1.9.0+cpu
+torch==1.9.0
```

Comment:
Why this?  Especially as I remember problems occurring in the past when we didn't explicitly specify `+cpu`.

Reply:
As I was able to see keeping "cpu" was causing issues, It seems torch has new updates.